app/packages/files/file.ts

```diff
@@ -65,6 +89,13 @@ export default abstract class File {
         return [];
     }
 
+    async getRevisions() {
```

Comment:
Should this be a task? We shouldn't need to restart it, but we probably want to show the pulser while we're loading information.